EC-Earth3-pr.yaml fails biascorrection output validation with negative values and NaNs

All ssps and historical are failing biascorrection validation for EC-Earth3's precipitation run.

From workflow run: https://argo.cildc6.org/archived-workflows/default/7fd8921a-2996-409a-9ec7-b8842c473787

Example of a logged error from the ssp370 run:

```
Validating gs://biascorrected-492e989a/stage/ScenarioMIP/EC-Earth-Consortium/EC-Earth3/ssp370/r1i1p1f1
/day/pr/gr/v20211216222735.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 33, in clear_memory_intensive_tests
    _test_negative_values(d, v)
  File "/opt/dodola/dodola/core.py", line 729, in _test_negative_values
    assert ds[var].where(ds[var] < -0.001).count() == 0, "there are negative values!"
AssertionError: there are negative values!
```

logged error for nans:
```
Validating gs://biascorrected-492e989a/stage/ScenarioMIP/EC-Earth-Consortium/EC-Earth3/ssp126/r1i1p1f1/day/pr/gr/v2021
1216222735.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 26, in clear_memory_intensive_tests
    _test_for_nans(d, v)
  File "/opt/dodola/dodola/core.py", line 634, in _test_for_nans
    assert ds[var].isnull().sum() == 0, "there are nans!"
AssertionError: there are nans!
```

Blocking progress on #266.

--- a/workflows/templates/qdm-preprocess.yaml
+++ b/workflows/templates/qdm-preprocess.yaml
@@ -154,7 +154,7 @@ spec:
           - name: out-zarr
             value: "{{ inputs.parameters.out-zarr }}"
       script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.12.0
+        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:dev
         command: [ python ]
         source: |
           import logging
@@ -165,7 +165,6 @@ spec:
 
           input_zarr = "{{ inputs.parameters.in-zarr }}"
 
-          wdf_corrected_zarr = "/workspace/wdf_corrected.zarr"
           dodola.services.correct_wet_day_frequency(
               "{{ inputs.parameters.in-zarr }}",
               process="pre",

--- a/workflows/templates/qualitycontrol-check-cmip6.yaml
+++ b/workflows/templates/qualitycontrol-check-cmip6.yaml
@@ -40,7 +40,7 @@ spec:
           - name: data
           - name: time
       script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.13.0
+        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:dev
         command: [python]
         source: |
           # Running this as a script rather than a dodola command as workaround to 

